mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -133,7 +134,10 @@ class CrossValidator @Since("1.2.0") (@Since("1.4.0") override val uid: String)
     logInfo(s"Best cross-validation metric: $bestMetric.")
     val bestModel = est.fit(dataset, epm(bestIndex)).asInstanceOf[Model[_]]
     instr.logSuccess(bestModel)
-    copyValues(new CrossValidatorModel(uid, bestModel, metrics).setParent(this))
+    val model = new CrossValidatorModel(uid, bestModel, metrics).setParent(this)
+    val summary = new TuningSummary(epm, metrics, bestIndex)
+    model.setSummary(Some(summary))
```

Comment:
If we want to just save the tuning summary in the model, Perhaps we can just discard the TuningSummary, and add a tuningSummary field in the models. Sounds good?